src/scss/themes/_small.scss

```diff
@@ -20,6 +20,11 @@
 		@include oGridRespondTo($until: M) { // Mobile screens only
 			display: none;
 		}
+
+		//hack because IE11 and flexbox aren't friends
+		.top-stories__freeform-twocolumns & {
```

Comment:
I don't think the added class name is required, the rule may simply be appended to the others.

Alternatively remove `flex-flow: row wrap;` as I can't see why it would be required by this variation.